app/services/current-recognition.js

```diff
@@ -2,10 +2,13 @@ import Service, { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 export default class CurrentRecognitionService extends Service {
+  items = ['College van burgemeester en schepenen', 'Andere'];
```

Comment:
It seems we're creating the hardcoded list twice. We can probably always reference this one, although I'm not sure why there is a dedicated `CurrentRecognition` service in the first place.